contracts/test-utils/src/web3_wrapper.ts

```diff
@@ -40,7 +40,7 @@ export const txDefaults = testProvider === ProviderType.Ganache ? ganacheTxDefau
 const gethConfigs = {
     shouldUseInProcessGanache: false,
     rpcUrl: 'http://localhost:8501',
-    shouldUseFakeGasEstimate: false,
+    shouldUseFakeGasEstimate: true,
```

Comment:
For some reason, the gas estimator in the latest version of Geth cannot estimate gas correctly when deploying our smart contracts. We've had some issues in the past but on the latest version it basically never works correctly. As a workaround we decided to disable it and just use the fake gas estimator.